descriptors/ModuleDescriptor-template.json

```
@@ -74,7 +74,7 @@
   "requires" : [
     {
       "id" : "source-manager-job-executions",
-      "version" : "2.0"
+      "version" : "3.0"
```

As mod-copycat is not using the end point that was changed, it will still support 2.0.. 
```suggestion
      "version" : "2.0 3.0"
```

---

updated dependency, thanks!